std/conv.d

```diff
@@ -5692,7 +5692,7 @@ auto toChars(ubyte radix = 10, Char = char, LetterCase letterCase = LetterCase.l
             enum SHIFT = 4;
         else
             static assert(0);
-        struct Result
+        static struct Result
```

Comment:
OK, I didn't know what the `static` keyword meant in this context or why your tests needed it, got it now.